### PR TITLE
[Enhancement] Improve broker load error message

### PR DESCRIPTION
**Pull request by dirtysalt**
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

This PR is to refactor broker load error message.

When broker load and error message type is `ETL_QUALITY_UNSATISFIED`, there is a extra field `TrackingSQL` to see the detailed error message, but many users don't know this. So I add a message "You can find detailed error message from running `TrackingSQL`" at the error message `ErrorMsg` field.

image

And in detailed error message, I rephrase it a little bit.

image

```
// This one means it failes to fill a csv field.
Error: The field (name = pair, pos = 2) is out of range. Type: ARRAY<VARCHAR(65533)>, Value: [ "1", "2" ]. Row: 6817;a;"[ ""1"", ""2"" ]"

// This one means value count does not match column count.
Error: Value count does not match column count: expected = 3, actual = 5. Column delimiter: 59, Row delimiter: 10. Row: 6418;a\b;"[ ""a\\b"", ""c\""d"" ]"
6419;"c""d";
```

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: be/src/exec/csv_scanner.cpp [hunk 1]
```diff
@@ -39,6 +39,26 @@ std::string string_2_asc(const std::string& input) {
     return output;
 }
 
+static std::string make_column_count_not_matched_error_message(int expected_count, int actual_count,
+                                                               CSVParseOptions& parse_options) {
+    std::stringstream error_msg;
+    error_msg << "Value count does not match column count: "
+              << "expected = " << expected_count << ", actual = " << actual_count << ". "
+              << "Column delimiter: " << string_2_asc(parse_options.column_delimiter) << ", "
```
Comment (jaogoy) [inline be/src/exec/csv_scanner.cpp:47]: Print 'Column separator' to match the parameters `column_separator` and `row_delimiter`.

Comment (jaogoy) [reply be/src/exec/csv_scanner.cpp:47]: And, can you print the `column_separator` using `\t`, `\n`, and hexadecimal code like `0x0D`, It's more readable than decimal code `9`, `10`.

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 14 / 38 (36.84%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/exec/csv_scanner.cpp|14|38|36.84%|[28, 42, 43, 56, 58, 330, 331, 333, 337, 374, 423, 425, 426, 429, 430, 431, 436, 437, 440, 456, 457, 458, 461, 462]|

File: be/src/exec/csv_scanner.cpp (hunk 1 above)
Comment (dirtysalt) [reply be/src/exec/csv_scanner.cpp:47]: fixed.

**Review by jaogoy — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by stephen-shelby — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Pull request merged**